docs/integrations/data-ingestion/clickpipes/object-storage/google-cloud-storage/01_overview.md

```diff
@@ -0,0 +1,142 @@
+---
+sidebar_label: 'Overview'
+description: 'Seamlessly connect your object storage to ClickHouse Cloud.'
+slug: /integrations/clickpipes/object-storage/gcs/overview
+sidebar_position: 1
+title: 'Integrating Google Cloud Storage with ClickHouse Cloud'
+---
+
+import cp_iam from '@site/static/images/integrations/data-ingestion/clickpipes/object-storage/google-cloud-storage/cp_iam.png';
```

Comment:
@morsapaes it's complaining about this image being missing from the google-cloud-storage directory. Does this need to be `cp_iam.png` from the amazon-s3 folder rather?